### PR TITLE
Form-169/Checkbox button

### PR DESCRIPTION
**Pull request by AkihitoVM**
base styling, need to fix tailwind styles, without tooltip, without responsiveness

### CODE REVIEW

**Comment by AkihitoVM** [comment]
Probable we should use grouped checkbox for RJSF, pls check styling for grouped checkbox ( max width of each container)

File: next/components/forms/SingleCheckbox.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        </div>
+      )}
+      <label htmlFor={rest.value} className={containerStyle}>
+        <VisuallyHidden>
+          <input id={rest.value} {...inputProps} {...focusProps} ref={ref} />
+        </VisuallyHidden>
+        <div className={checkboxStyle}>
+          {isSelected && (
+            <svg
+              className={cx('', {
```
Comment (mpinter) [inline next/components/forms/SingleCheckbox.tsx:99]:
```suggestion
              className={cx({
```

File: next/components/forms/SingleCheckbox.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+            >
+              <path
+                d="M5.49999 9.47504L2.02499 6.00004L0.845825 7.17921L5.49999 11.8334L15.5 1.83337L14.3208 0.654205L5.49999 9.47504Z"
+                fill="white"
+              />
+            </svg>
+          )}
+          {rest.isIndeterminate && (
+            <svg
+              className={cx('', {
```
Comment (mpinter) [inline next/components/forms/SingleCheckbox.tsx:115]:
```suggestion
              className={cx({
```

**Review by mpinter — APPROVED** [review]
🚢

File: next/components/forms/CheckboxGroup.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  label?: string
+}
+
+const CheckboxGroup = (props: CheckBoxGroupBase) => {
+  const { children, label } = props
+  const state: CheckboxGroupState = useCheckboxGroupState(props)
+  const { groupProps, labelProps } = useCheckboxGroup(props, state)
+
+  return (
+    <div {...groupProps} className="">
```
Comment (mpinter) [inline next/components/forms/CheckboxGroup.tsx:18]:
```suggestion
    <div {...groupProps}>
```

**Pull request merged**